app/src/test/java/org/simple/clinic/registration/pin/RegistrationPinScreenLogicTest.kt

```diff
@@ -53,8 +53,9 @@ class RegistrationPinScreenLogicTest {
     uiEvents.onNext(RegistrationPinDoneClicked())
 
     // then
-    verify(userSession).saveOngoingRegistrationEntry(ongoingRegistrationEntry.withPin(input))
-    verify(uiActions).openRegistrationConfirmPinScreen()
+    val entryWithPin = ongoingRegistrationEntry.withPin(input)
```

Comment:
Is this the common test for both screens `RegistrationPinScreen` and `RegistrationConfirmPinScreen` ?

Reply:
No, it's not a common test. But since we are passing it to the screen key of `RegistrationConfirmPinScreen`, we have to modify the UI action of `RegistrationPinScreen` to accept `OngoingRegistrationEntry` as a parameter, so the test has to account for this as well.